src/bootstrap-node.ts

```diff
@@ -7,7 +7,7 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { fileURLToPath } from 'url';
 import { createRequire } from 'node:module';
-import type { IProductConfiguration } from './vs/base/common/product';
+import type { IProductConfiguration } from './vs/base/common/product.ts';
```

Comment:
How come this is not `.js`?

Reply:
Good catch. It should have been `.js`. Will see if I can configure TS to generate an error in this case too